src/main/java/net/datafaker/providers/base/Azure.java

```diff
@@ -0,0 +1,147 @@
+package net.datafaker.providers.base;
+
+
+/**
+ * Generates data for Azure services. This is based on the Azure best practices of naming conventions:
+ * <a href="https://learn.microsoft.com/en-us/azure/cloud-adoption-framework/ready/azure-best-practices/resource-naming">...</a>
```

Comment:
Small mistake here, I think you used IntelliJ to autofix the urls? (I had a similar issue.)

Reply:
Yes, Intellij :o) Thanks for fixing it